handlers/elasticsearch-templates.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+- name: Ensure elasticsearch is started
+  service: name={{instance_init_script | basename}} state=started enabled=yes
+
+- name: Wait for elasticsearch to startup
+  wait_for: port={{es_api_port}} delay=10
+
+- name: Get template files
+  shell: find . -maxdepth 1 -type f | sed "s#\./##" | sed "s/.json//" chdir=/etc/elasticsearch/templates
```

Comment:
You should use the [find module](http://docs.ansible.com/ansible/find_module.html) available in ansible 2. 

There is a ``patterns`` option if you want to filter out *.json file.